activiti-cloud-notifications-graphql-service/dependencies/pom.xml

```diff
@@ -10,6 +10,9 @@
   <name>Activiti Cloud Notifications :: GraphQL Dependencies (Bill Of Materials)</name>
   <artifactId>activiti-cloud-notifications-graphql-dependencies</artifactId>
   <packaging>pom</packaging>
+  <properties>
+    <querydsl.version>5.0.0</querydsl.version>
+  </properties>
```

Comment:
It's better to centralize the changes in a single place if it's possible. Have you tried to add the changes to [activiti-cloud-service-common-dependencies](https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-service-common/activiti-cloud-service-common-dependencies/pom.xml)? It should update it for both audit, query and notifications.

Reply:
I totally agree: it's way better to centralise this changes. However, activiti-cloud-service-common-dependencies it's not  in the inheritance path of query, audit and notifications projects. Accordingly, we'd not update the querydsl version.
I've tested that with the dependency:tree.
Should we re-think that inheritance structure?

Reply:
Ok, in this case, it's better to keep it like that. Re-thinking the inheritance structure would take time, and this is a temporary workaround as we should be able to remove it once spring boot references the new version of querydsl.